llvm/include/llvm/IR/DataLayout.h

```diff
@@ -159,16 +146,19 @@ class DataLayout {
   /// well-defined bitwise representation.
   SmallVector<unsigned, 8> NonIntegralAddressSpaces;
 
-  /// Attempts to set the alignment of the given type. Returns an error
-  /// description on failure.
-  Error setAlignment(AlignTypeEnum AlignType, Align ABIAlign, Align PrefAlign,
-                     uint32_t BitWidth);
+  // Attempts to set the specification for the given type.
+  // Returns an error description on failure.
```

Comment:
Use `///` here and elsewhere.

Reply:
Shouldn't it be used for public interfaces only?

Reply:
It does not matter whether the interface is public or not. `//` would only be appropriate if you want to leave an implementation note (e.g. a TODO).

Reply:
If you're documenting an item, it should always be a doxygen comment. This can be any kind of class member, but also e.g. a static function in a source file. (Our default doxygen config does not generate private member documentation, but peculiarly it does generate static item docs.)